src/blocks/Banner/Banner.tsx

```diff
@@ -19,9 +19,11 @@ export const Banner: React.FC<BannerProps> = ({
     image,
     paddingTop,
     paddingBottom,
+    qa,
     ...content
 }) => {
     const contentStyle: Record<string, string> = {};
+    const qas = getCommonQa(qa, ['image-container']);
```

Comment:
maybe use different name, like `commonQA` ?

Reply:
I named them after variable in PC PR https://github.com/gravity-ui/page-constructor/pull/537#discussion_r1314650109
`qaAttributes`